src/database/seeds/testimonials.ts

```diff
@@ -3,10 +3,11 @@ import faker from 'faker'
 import database from '../../config/database';
 
 export async function seed(): Promise<void> {
+  const data = []
+
   for (let index = 0; index < 10; index++) {
```

Comment:
how about giving a parameter to this function instead of 10 as the default number of the inserted rows.

Reply:
how do you mean ?

Reply:
```ts
export async function seed(numOfRows: number): Promise<void> {
  for (let index = 0; index < numOfRows; index++) {
```

Reply:
This is not a method as usual, indeed the behavior of writing a seeder is like this